spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -5,409 +5,411 @@
+        )
 
-      expect(response.body).to have_content('Please enter a valid email')
-      expect(response.body).to_not have_content('has already been taken')
-      expect(enqueued_jobs).to eq []
-      expect(performed_jobs).to eq []
+        expect(response.body).to have_content('Please enter a valid email')
+        expect(response.body).to_not have_content('has already been taken')
+        expect(enqueued_jobs).to eq []
+        expect(performed_jobs).to eq []
+      end
     end
-  end
 
-  context "user updates profile with invalid mobile and another user's email" do
-    render_views
+    context "user updates profile with invalid mobile and another user's email" do
+      render_views
 
-    it 'displays error about invalid email', email: true do
-      sign_in(user)
-      put(
-        :update,
-        update_user_profile_form: attrs_with_already_taken_mobile.merge!(
-          email: second_user.email, mobile: '703'
+      it 'displays error about invalid email', email: true do
+        sign_in(user)
+        put(
+          :update,
+          update_user_profile_form: attrs_with_already_taken_mobile.merge!(
+            email: second_user.email, mobile: '703'
+          )
         )
-      )
 
-      expect(flash).to be_empty
-      expect(response.body).to have_content('number is invalid')
-      expect(response.body).to_not have_content('has already been taken')
-      expect(last_email).to be_nil
+        expect(flash).to be_empty
+        expect(response.body).to have_content('number is invalid')
+        expect(response.body).to_not have_content('has already been taken')
+        expect(last_email).to be_nil
+      end
     end
-  end
 
-  context 'blank new password' do
-    render_views
+    context 'blank new password' do
+      render_views
 
-    it 'does not change password' do
-      sign_in(user)
-      put :update, update_user_profile_form: attrs_with_blank_new_password
+      it 'does not change password' do
+        sign_in(user)
+        put :update, update_user_profile_form: attrs_with_blank_new_password
 
-      expect(user.reload.encrypted_password).to eq old_encrypted_password
-      expect(response).to redirect_to edit_user_registration_url
+        expect(user.reload.encrypted_password).to eq old_encrypted_password
+        expect(response).to redirect_to edit_user_registration_url
+      end
     end
-  end
 
-  context 'invalid current password' do
-    render_views
+    context 'invalid current password' do
+      render_views
 
-    it 'displays invalid password error' do
-      sign_in(user)
-      put :update, update_user_profile_form: attrs_with_invalid_current_password
+      it 'displays invalid password error' do
+        sign_in(user)
+        put :update, update_user_profile_form: attrs_with_invalid_current_password
 
-      expect(response.body).to have_content('is invalid')
+        expect(response.body).to have_content('is invalid')
+      end
```

Comment:
All of the changes above this line was just me wrapping all of the specs above inside a new `describe` block because the `let` statements don't apply to the new spec I added below.